core/android/database/src/main/java/com/vgleadsheets/database/android/UserContentDatabaseVersions.kt

```diff
@@ -6,7 +6,8 @@ object UserContentDatabaseVersions {
     /**
      * - Added "offline_song" table.
      * - Added "offline_game" table.     // TODO
-     * - Added "offline_composer" table. // TODO
+     * - Added "offline_composer" table.
```

Comment:
Creating a third database version here is acceptable, though not necessary. But if we're going to do it, we should rewrite the comments to reflect that.

(If we're going to have a new version for all three models, we probably don't need the comment at all, given how they're just restating the name of the version variable.)

Reply:
Comment removed